fabfile.py

```diff
@@ -24,6 +24,11 @@ def verify():
     test()
 
 
+def runserver():
+    local('python src-backend/manage.py syncdb --noinput')
```

Comment:
uhhh, I think it just ensures that we don't have to type anything. I'm just following the rest of the fab commands.